src/test/ui/const-generics/issues/issue-62456.rs

```diff
@@ -0,0 +1,9 @@
+#![feature(const_generics)]
```

Comment:
File should not be marked as executable.

Reply:
I've corrected all the suggested items. Thank you for your suggestion.